pkgs/development/compilers/osl/default.nix

```diff
@@ -2,13 +2,13 @@
 , lib
 , fetchFromGitHub
 , cmake
-, clang
-, libclang
+, clang_15
```

Comment:
https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults

its preferred to override in all-packages

Reply:
Makes sense, makes the diff simpler too.